test/WebSites/FormatterWebSite/Startup.cs

```diff
@@ -19,12 +19,12 @@ public void ConfigureServices(IServiceCollection services)
 
                 options.InputFormatters.Add(new StringInputFormatter());
             })
-            .AddXmlDataContractSerializerFormatters();
+            .AddXmlDataContractSerializerFormatters()
+            .SetCompatibilityVersion(CompatibilityVersion.Latest);
```

Comment:
Should bump up all functional tests to run with `Latest` i.e. change `MvcTestFixture<TStartup>` instead of making this one-off update

Reply:
Causes a bunch of endpoint routing tests to fail.

Reply:
Please open a bug about that. Our functional testing should focus on `Latest` and something's wrong if failures occur when everything's turned on. Of course, we may now need to add a few (very few) 2.1-specific tests.

Reply:
https://github.com/aspnet/Mvc/issues/8274